varcode/effect_collection.py

```diff
@@ -143,8 +143,7 @@ def drop_silent_and_noncoding(self):
         """
         Create a new EffectCollection containing only non-silent coding effects
         """
-        return self.filter(
-            lambda effect: isinstance(effect, NonsilentCodingMutation))
+        return self.filter(lambda effect: effect.modifies_protein_sequence)
```

Comment:
Yeah, that was my understanding of the changes as well; just wondering if there are classes that need to be modified, etc.